RX.h

```diff
@@ -9,6 +9,11 @@ uint32_t lastPacketTimeUs = 0;
 uint32_t lastRSSITimeUs = 0;
 uint32_t linkLossTimeMs;
 
+uint32_t hopInterval = 0;
+uint32_t hopTimeout = 0;
+uint32_t hopTimeoutSlow = 0;
+uint32_t RSSI_timeout = 0;
```

Comment:
isn't it already in CC? Do you mean capitalizing the first letter? 

For RSSI_timeout, I believe I copied the format from some other 'RSSI_*' variable that I saw somewhere in the code (I'd have to go back and check, can't remember right now).